### ISSUE TITLE
kernel must be 'precomputed' or callable, is precomputed.

### ISSUE THREAD

**Issue by nicobrb** [pyRiemann/pyRiemann#271]
Given the model 
```
model = process.SVC(metric="logeuclid", kernel_fct="precomputed", C=0.1,
                                    shrinking=True, cache_size=1024,
                                    class_weight="balanced")
```

Then calling

`model.fit(X, y)`

returns 
`kernel must be 'precomputed' or callable, is precomputed.`

This is due to this piece of code in `pyriemann.classification.py`:

![image](https://github.com/pyRiemann/pyRiemann/assets/39137998/4b14584c-2840-4d87-8880-5c48b30b707b)

Which clearly doesn't check where `kernel_fct = "precomputed"`, but only if it's callable or None.

**Comment by qbarthelemy** [comment]
Good catch! This problem will be solved with #272.

In the meantime you can use `kernel_fct=None` to achieve the same behavior.

**Comment by qbarthelemy** [comment]
Closed with #272
